kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java

```diff
@@ -93,12 +90,11 @@ public void updateConnector() {
                 .openConnector(CONNECTOR_FOR_UPDATE);
         pages.connectorsView.connectorIsVisibleOnOverview();
         pages.connectorsView.openEditConfig()
-                .updConnectorConfig(FileUtils.getResourceAsString("config_for_update_connector.json"));
+                .updConnectorConfig(Objects.requireNonNull(FileUtils.getResourceAsString("config_for_update_connector.json")));
```

Comment:
no need to place here Objects.requireNonNull because getResourceAsString() doesn't have return null option now

Reply:
Reverted